### PR TITLE
Some more inductor fixes for symbolic shapes

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #92878
* __->__ #92867

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/config.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ class triton:
     # should we give different names to kernels
     ordered_kernel_names = False
     # should we put op names in kernel names
-    descriptive_kernel_names = True
+    descriptive_kernel_names = False
```
Comment (ezyang) [inline torch/_inductor/config.py:175]: lol

File: benchmarks/dynamo/common.py [hunk 2]
```diff
@@ -1837,6 +1837,7 @@ def run(runner, args, original_dir=None):
     if args.dynamic_shapes:
         torch._dynamo.config.dynamic_shapes = True
         torch._functorch.config.use_dynamic_shapes = True
+        torch._inductor.config.dynamic_shapes = True
```
Comment (ezyang) [inline benchmarks/dynamo/common.py:1840]: You're going to wanna update dynamic=True implementation too which monkeypatches these configs

**Review by ezyang — APPROVED** [review]

File: benchmarks/dynamo/common.py (hunk 2 above)
Comment (Chillee) [reply benchmarks/dynamo/common.py:1840]: hmm... that is leaking Inductor details up into Dynamo though. I think we just need to refactor this to have Inductor pull this flag down from Dynamo. cc: @voznesenskym

**Pull request closed without merging**